Packs/FortiGate/Integrations/FortiGate/FortiGate.py

```diff
@@ -0,0 +1,1267 @@
+            'Subnet': subnet,
+            'StartIP': address.get('start-ip'),
+            'EndIP': address.get('end-ip')
+        })
+
+    context['Fortigate.Address(val.Name && val.Name === obj.Name)'] = addresses_context
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': contents,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('FortiGate addresses', contents),
+        'EntryContext': context
+    })
+
+
+def get_addresses_request(address, name):
+    uri_suffix = 'cmdb/firewall/address/' + name
+    params = {
+        'vdom': address
+    }
+    response = http_request('GET', uri_suffix, params)
+    # Different structure if we choose all domains
+    if address == '*':
+        return response[0].get('results')
+    return response.get('results')
+
+
+def create_address_command():
+    contents = []
+    context = {}
+    address_context = []
+    address_name = demisto.args().get('name', '')
+    address = demisto.args().get('address', '')
+    mask = demisto.args().get('mask', '')
+    fqdn = demisto.args().get('fqdn', '')
+
+    if fqdn and address:
+        return_error("Please provide only one of the two arguments: fqdn or address")
+
+    create_address_request(address_name, address, mask, fqdn)
+
+    if address:
+        contents.append({
+            'Name': address_name,
+            'IPAddress': address
+        })
+        address_context.append({
+            'Name': address_name,
+            'IPAddress': address
+        })
+    elif fqdn:
+        contents.append({
+            'Name': address_name,
+            'FQDN': fqdn
+        })
+        address_context.append({
+            'Name': address_name,
+            'FQDN': fqdn
+        })
+
+    context['Fortigate.Address(val.Name && val.Name === obj.Name)'] = address_context
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': contents,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('FortiGate address ' + address_name + ' created successfully', contents),
+        'EntryContext': context
+    })
+
+
+def create_address_request(address_name, address, mask, fqdn):
+    uri_suffix = 'cmdb/firewall/address/'
+    if does_path_exist(uri_suffix + address_name):
+        return_error('Address already exists.')
+    subnet = address + " " + mask
+    if address:
+        payload = {
+            'name': address_name,
+            'subnet': subnet
+        }
+    elif fqdn:
+        payload = {
+            'name': address_name,
+            "type": "fqdn",
+            "fqdn": fqdn
+        }
+    result = http_request('POST', uri_suffix, {}, json.dumps(payload))
+    return result
+
+
+def delete_address_command():
+    contents = []
+    context = {}
+    address_context = []
+    name = demisto.args().get('name', '')
```

Comment:
Is there any usecase that a user would like to delete multiple addresses at the same time?
Is it worth making this argument as list and delete multiple addresses with one command?